specification/_json_spec/ingest.put_geoip_database.json

```diff
@@ -7,8 +7,7 @@
     "stability": "stable",
     "visibility": "public",
     "headers": {
-      "accept": ["application/json"],
-      "content_type": ["application/json"]
+      "accept": ["application/json"]
```

Comment:
This needs to go away before merging. We just need to wait for a more recent snapshot that `8.16.0-3a37a73c` in https://snapshots.elastic.co/latest/master.json.